libc/src/__support/threads/linux/raw_mutex.h

```diff
@@ -37,7 +44,8 @@ class RawMutex {
   LIBC_INLINE_VAR static constexpr FutexWordType LOCKED = 0b01;
   LIBC_INLINE_VAR static constexpr FutexWordType IN_CONTENTION = 0b10;
 
-  LIBC_INLINE FutexWordType spin(uint_fast32_t spin_count) {
+private:
+  LIBC_INLINE FutexWordType spin(int spin_count) {
```

Comment:
This changes the signedness of `spin_count`. Do we want to use `unsigned` instead?